python/Ice/filesystem/server/m_file.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) ZeroC, Inc.
+
+from Filesystem import File
+import Ice
+
+class MFile(File):
+    """
+    Provides an in-memory implementation of the Slice interface File.
+    """
+
+    def __init__(self, name):
+        """
+        Initializes a MFile with the given name.
+
+        Parameters
+        ----------
+        name : str
+            The name of the file.
+        """
+        self._name = name
+        self._lines = []
+
+
+    def name(self, current: Ice.Current) -> str:
```

Comment:
A big part of this demo is to showcase implementation reuse with multiple inheritance - with a reusable MNode, like in C++. Can you do that in Python too? We can't do it in C# since we map Disp to classes and C# has no multiple inheritance for classes.

Reply:
Reworked to use multiple inheritance.